### PR TITLE
fix readme en

### PR DESCRIPTION
**Pull request by littletomatodonkey**
fix doc and english readme

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -2,152 +2,313 @@
...
-Taking the ImageNet1k dataset as an example, the following figure shows the SSLD knowledge distillation method framework. The key points of the method include the choice of teacher model, loss calculation method, iteration number, use of unlabeled data, and ImageNet1k dataset finetune. For detailed introduction and experiments, please refer to [**knowledge distillation tutorial**](https://paddleclas-en.readthedocs.io/en/latest/advanced_tutorials/distillation/distillation_en.html)
-
-<div align="center">
-<img
-src="./docs/images/distillation/ppcls_distillation_s.jpg" width="700">
-</div>
 
-### Data augmentation
+**Recent update**
+- 2020.07.17 Add `HRNet_W48_C_ssld` pretrained model, whose Top-1 Acc on ImageNet1k dataset reaches 83.62%. Add `ResNet34_vd_ssld` pretrained model, whose Top-1 Acc on ImageNet1k dataset reaches 79.72%.
```
Comment (TingquanGao) [inline README_en.md:11]: 时间线是否可以保值一致，会更好？

File: README_en.md [hunk 2]
```diff
@@ -2,152 +2,313 @@
...
+- 2020.06.17 Add English documents.
+- 2020.06.12 Add support for training and evaluation on Windows or CPU.
+- 2020.05.17 Add support for mixed precision training.
+- [more](./docs/zh_CN/update_history.md)
 
-For a certain image classification task, data augmentation is a commonly used regularization method, which can effectively improve the effect of image classification, especially for scenarios where the data is insufficient or the network is large. Commonly used data augmentation can be divided into 3 categories, `transformation`, `cropping` and `aliasing`, as shown below. The image transformation refers to performing some transformations on the entire image, such as AutoAugment and RandAugment. The image cropping refers to the transformation of the image to block some areas in a certain way, such as CutOut, RandErasing, HideAndSeek and GridMask. Image aliasing refers to the transformation of multiple images to alias a new image, such as Mixup and Cutmix.
 
+## Rich model zoo
```
Comment (TingquanGao) [inline README_en.md:20]: 该标题下没有内容？标题是否需要删除？

File: README_en.md [hunk 3]
```diff
@@ -2,152 +2,313 @@
...
 
+## Features
 
-- TODO
-- [ ] Application of PaddleClas in OCR tasks.
-- [ ] Application of PaddleClas in face detection and recognition tasks.
+- Rich model zoo. Based on the ImageNet1k classification dataset, PaddleClas provides 24 series of classification network structures and training configurations, 122 models' pretrained weights and their evaluation metrics.
 
+- SSLD Knowledge Distillation. Based on this SSLD distillation strategy, the accuracy of the distillation model is generally increased by more than 3%.
```
Comment (TingquanGao) [inline README_en.md:27]: distillation model？
distilled model

File: README_en.md [hunk 4]
```diff
@@ -2,152 +2,313 @@
...
 
+- Data augmentation: PaddleClas provides detailed introduction of 8 data augmentation algorithms such as AutoAugment, Cutout, Cutmix, code reproduction and effect evaluation in a unified experimental environment.
 
-## Industrial-grade application deployment tools
+- Pretrained model with 100,000 categories: Baidu self-developed and open sourced the `ResNet50_vd` model trained on a 100,000-category dataset. In some practical scenarios, the  accuracy based on the pretrained weights can be increased by up to 30%.
```
Comment (TingquanGao) [inline README_en.md:31]: "Baidu self-developed and open sourced the `ResNet50_vd` model trained on a 100,000-category dataset"?
The `ResNet50_vd` model Baidu self-developed and open sourced trained on a 100,000-category dataset.

File: README_en.md [hunk 5]
```diff
@@ -2,152 +2,313 @@
...
-- TensorRT inference
-- Paddle-Lite
-- Paddle-Serving
-- Model quantization
-- Multi-machine training
-- Paddle Hub
+- A variety of inference and deployment solutions, including TensorRT inference, Paddle-Lite inference, model service deployment, model quantification, Paddle Hub, etc.
 
-## Competition support
+- Support Linux, Windows, MacOS and other systems.
```
Comment (TingquanGao) [inline README_en.md:37]: MacOS
macOS

File: README_en.md [hunk 6]
```diff
@@ -2,152 +2,313 @@
...
+    - C++ inference (coming soon)
+    - [Serving deployment](./docs/en/extension/paddle_serving_en.md)
+    - Mobile (coming soon)
+    - [Model Quantization and Compression](docs/en/extension/paddle_quantization_en.md)
+- Advanced tutorials
+    - [Knowledge distillation](./docs/en/advanced_tutorials/distillation/distillation_en.md)
+    - [Data augmentation](./docs/en/advanced_tutorials/image_augmentation/ImageAugment_en.md)
+- Applications
+    - [Transfer learning](./docs/en/application/transfer_learning_en.md)
+    - [pretrained model with 100,000 categories](./docs/en/application/transfer_learning_en.md)
```
Comment (TingquanGao) [inline README_en.md:70]: Pretrained model with 100,000 categories

Comment (littletomatodonkey) [reply README_en.md]: fixe, thx

File: README_en.md (hunk 5 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed, thx

File: README_en.md (hunk 4 above)
Comment (littletomatodonkey) [reply README_en.md]: fix, thx

File: README_en.md (hunk 3 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed, thx

File: README_en.md (hunk 2 above)
Comment (littletomatodonkey) [reply README_en.md]: done, thx

File: README_en.md (hunk 1 above)
Comment (littletomatodonkey) [reply README_en.md]: fixed, thx

**Pull request merged**